### ISSUE TITLE
Skype Ads Pose Accessibility Problems In Latest Version Of Skype For NVDA

### ISSUE THREAD

**Issue by Elshara** [nvaccess/nvda#5975]
When navigating group and private chat windows, a new advertisement appears in Skype which appears to be badly coded and NVDA is unable to exit out of it but by tabbing an unknown amount of times, it causes sever lag and jumping issues for the cursor to land on a focused point within the Skype interface.

**Comment by Adriani90** [comment]
@Elshara, @k-kolev1985, @ruifontes can anybody reproduce this in last skype version? At least on Skype UWP I cannot reproduce this problem.

**Comment by k-kolev1985** [comment]
No, I cannot reproduce this in Skype 8 or Skype for Windows 10 from the Microsoft Store. Come to think of it, I haven't noticed it in a long time in Skype 7 as well.
